### PR TITLE
Add gallerypanel a1 to implement 1

### LINKED ISSUE #2: There should be a way to add more content to the picture in the gallery
**Is your feature request related to a problem? Please describe.**
It really pisses me offf when i see a picture without any information whatsoever. It is really hollow like that.

**Describe the solution you'd like**
I want to add a information panel under the picture .

**Additional context**
Ref: https://test.angelo.world/

### PR DESCRIPTION
**Pull request by petschna**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [ ] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no way to have a gallery information panel. #2 

**What is the new behavior (if this is a feature change)?**
- Now there is a way to have a gallery information panel. Closes #2

**Other information**:
- Ref: https://test.angelo.world/

### CODE REVIEW

File: src/components/molecules/SectionContainer/index.jsx [hunk 1]
```diff
@@ -39,6 +39,8 @@ let SectionContainer = ({
   title = title ? <h2 className="mb-3">{title}</h2> : "";
   header = header ? <h4 className="mb-2">{header}</h4> : "";
 
+  // want to add a information panel for gallery
+
```
Comment (petute) [inline src/components/molecules/SectionContainer/index.jsx]: Please add this panel

**Review by petute — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by petute — APPROVED** [review]
Should be ok to merge.

**Review by petschna — COMMENTED** [review]
Should be ok to merge

**Review by kleberbaum — APPROVED** [review]
should be ok to merge.

**Pull request merged**